Digital_Fusion/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,51 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-top: 2px solid rgba(75, 5, 75, 0.1);
+  max-width: 100%;
+  background-color: #000;
+  box-sizing: border-box;
+  font-family: "Alegreya";
```

Comment:
It's important to always provide **fallback** and **generic** font name for `font-family`, like that: font-family: 'Inter', Arial, sans-serif; 
https://developer.mozilla.org/en/docs/Web/CSS/font-family

Font names are just for an example, you have your own font names and need to set fallback fonts based on it. For example, `Noto Serif` and `Inter` belong to different types, and they need different fallback fonts.
https://www.w3schools.com/css/css_font.asp 

Please, make this correction for **all the places** where you use `font-family`